core/commands/blame.py

```diff
@@ -376,52 +378,33 @@ def show_file_at_commit(self, from_line=False):
             "commit_hash": commit_hash,
             "filepath": self.file_path,
             "lineno": self.find_lineno(),
-            "lang" : self.view.settings().get('git_savvy.original_syntax', None)
+            "lang": self.view.settings().get('git_savvy.original_syntax', None)
         })
 
     def pick_new_commit(self):
         self.view.run_command("gs_blame_pick_commit", {
             "commit_hash": self.view.settings().get("git_savvy.commit_hash"),
         })
 
+
 class GsBlamePickCommitCommand(TextCommand, GitCommand):
 
     def run(self, *args, commit_hash=None):
         sublime.set_timeout_async(lambda: self.run_async(self.file_path), 0)
 
     def run_async(self, file_path):
-        settings = self.view.settings()
-        settings.set("git_savvy.commit_hash_old", settings.get("git_savvy.commit_hash"))
```

Comment:
if the panel is cancelled, the commit will set to `self.commit_hash`, which is the original commit (line 407).